<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastrarCompra</class>
 <widget class="QDialog" name="cadastrarCompra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastra Compra</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>319</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome do produto</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="consultaNomeProduto"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="botaoConsulta">
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quantidade Comprada</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="botaoUpdate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="novaQuantidade">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="retornoProduto">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
